themes/src/main/resources/theme/keycloak-preview/account/resources/README.md

```diff
@@ -0,0 +1,48 @@
+New Account Console
+========
+
+The new Account Console uses PatternFly 4 and React.
+
+Here is what you need to know in order to do development on the new console.
+
+
+Building
+--------
+First, you must install node and npm.
+
+To build and lint the application:
+
+`npm run install`
+
+If your IDE doesn't handle dynamic TypeScript transpilation, use the watch script:
+
+`npm run watch`
+
+If you want to run lint by itself**:
+
+`npm run lint`
```

Comment:
@ssilvert I can be wrong, but I believe the `lint` script is missing into the `package.json` file.

Reply:
I think you just need the latest commit with the lint stuff in it.

Reply:
@ssilvert that's strange, I have all the changes and I can see the `lint` script on master, but not on this PR. Maybe this PR needs a rebase?

Reply:
I don't see a reason to go to the trouble of rebasing the PR.  It's just documentation.  So once this PR is merged then the documentation will match the code.

Reply:
@abstractj ^^